test/functional/wallet_basic.py

```diff
@@ -661,5 +661,13 @@ def run_test(self):
         assert_array_result(tx["details"], {"category": "receive"}, expected_receive_vout)
         assert_equal(tx[verbose_field], self.nodes[0].decoderawtransaction(tx["hex"]))
 
+        self.log.info("Test send* RPCs with verbose=True")
+        address = self.nodes[0].getnewaddress("test")
+        txid_feeReason_one = self.nodes[2].sendtoaddress(address = address, amount = 10, verbose = True)
+        assert_equal(txid_feeReason_one["fee_reason"], "Fallback fee")
+        txid_feeReason_two = self.nodes[2].sendmany(dummy = '', amounts = {address: 10}, verbose = True)
+        assert_equal(txid_feeReason_two["fee_reason"], "Fallback fee")
```

Comment:
maybe add a case where you explicitly pass in `verbose = False` to cover that case too :+1:

Reply:
Added `verbose = False` case in test file